src/pretix/control/forms/organizer_forms/organizer_form.py

```diff
@@ -22,3 +27,153 @@ def clean_slug(self):
+        ),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    primary_contact_email = forms.EmailField(
+        label=_("Primary Contact Email"),
+        help_text=_(
+            "We will use this email address for all communication related to your contract and billing, "
+            "as well as for important updates about your account and our services."
+        ),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    company_or_organization_name = forms.CharField(
+        label=_("Company or Organization Name"),
+        help_text=_("Enter your organization’s legal name."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    address_line_1 = forms.CharField(
+        label=_("Address Line 1"),
+        help_text=_("Street address or P.O. box."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    address_line_2 = forms.CharField(
+        label=_("Address Line 2"),
+        help_text=_("Apartment, suite, unit, etc. (optional)."),
+        required=False,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    zip_code = forms.CharField(
+        label=_("Zip Code"),
+        help_text=_("Enter your postal code."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    city = forms.CharField(
+        label=_("City"),
+        help_text=_("Enter your city."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    country = forms.ChoiceField(
+        label=_("Country"),
+        help_text=_("Select your country."),
+        required=True,
+        choices=CachedCountries(),
+        initial="US",
+    )
+
+    preferred_language = forms.ChoiceField(
+        label=_("Preferred Language for Correspondence"),
+        help_text=_("Select your preferred language for all communication."),
+        required=True,
+    )
+
+    tax_id = forms.CharField(
+        label=_("Tax ID (e.g., VAT, GST)"),
+        help_text=_(
+            "If you are located in the EU, please provide your VAT ID. "
+            "Without this, we will need to charge VAT on our services and will not be able to issue reverse charge invoices."
+        ),
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+        required=False,
+    )
+
+    def __init__(self, *args, **kwargs):
+        self.organizer = kwargs.pop("organizer", None)
+        selected_languages = [
+            (code, name)
+            for code, name in settings.LANGUAGES
+            if code in self.organizer.settings.locales
+        ]
+        self.base_fields["preferred_language"].choices = selected_languages
```

Comment:
Why do you modify `self.base_fields` instead of `self.fields`?

Reply:
I use self.base_fields instead of self.fields because I want to modify the original field definitions before the form is initialized. Therefore, I use self.base_fields before calling super().__init__(*args, **kwargs). On the other hand, if you need to modify form fields after the form has been initialized, you should use self.fields. Thank you for reviewing. If you have any further questions, please let me know